core/DAPS/README.md

```diff
@@ -1,16 +1,212 @@
 # Dynamic Attribute Provisioning Service (DAPS)
 
+The infrastructure component "Digital Attribute Provisioning Service" (DAPS)
```

Comment:
"Digital Attribute Provisioning Service" -> should be "Dynamic Attribute Provisioning Service"

Reply:
Absolutely right. Wherever this digital came from...